benchmarks-mobile/protonano_app/app/src/main/java/io/grpc/benchmarks/BenchmarkResult.java

```diff
@@ -1 +1,30 @@
+package io.grpc.benchmarks;
 
+/**
+ * Created by davidcao on 6/3/16.
+ */
+public class BenchmarkResult {
+    public String name;
+    public int iterations;
+    public long elapsed;
+    public float mbps;
+    public long size;
+    public long compressedSize;
+
```

Comment:
Consider adding some comments to these fields to avoid confusion. For example, what's the unit for elapsed. Is it bits or bytes for mbps?